homeassistant/components/tplink/entity.py

```diff
@@ -20,10 +21,14 @@
 def async_refresh_after(
```

Comment:
Maybe we should rename this `async_wrap_exception_then_refresh` or have a separate wrapper.

Reply:
Sounds good to me, I don't think it's worth having a separate decorator for this purpose 👍